Makefile

```diff
@@ -19,19 +20,19 @@ GIT_COMMIT=$(shell git rev-parse --short HEAD)
 
 nginx-ingress:
 ifeq ($(BUILD_IN_CONTAINER),1)
-	$(DOCKER_BUILD_RUN) -e CGO_ENABLED=0 -e GO111MODULE=on -e GOFLAGS='-mod=vendor' $(GOLANG_CONTAINER) go build -installsuffix cgo -ldflags "-w -X main.version=${VERSION} -X main.gitCommit=${GIT_COMMIT}" -o /go/src/github.com/nginxinc/kubernetes-ingress/nginx-ingress
+	$(DOCKER_BUILD_RUN) -e CGO_ENABLED=0 -e GO111MODULE=on -e GOFLAGS=$(GOFLAGS) $(GOLANG_CONTAINER) go build -installsuffix cgo -ldflags "-w -X main.version=${VERSION} -X main.gitCommit=${GIT_COMMIT}" -o /go/src/github.com/nginxinc/kubernetes-ingress/nginx-ingress
```

Comment:
There is a small inconvenience:

if somebody needs to pass multiple flags, like
```
make ... GOFLAGS="flag1 flag2 flag3"
```
It would not work.
 They will have to do something like this:
```
make ... GOFLAGS="'flag1 flag2 flag3'"
```

Reply:
Thanks @pleshakov, I've pushed a change to allow to pass in multiple flags, e.g.:
`make nginx-ingress GOFLAGS="-mod=vendor -gcflags=-trimpath=$(go env GOPATH) -asmflags=-trimpath=$(go env GOPATH)"`